drivers/sensor/vl53l0x/vl53l0x.c

```diff
@@ -263,9 +263,79 @@ static int vl53l0x_channel_get(const struct device *dev, enum sensor_channel cha
 	return 0;
 }
 
+static int vl53l0x_get_sampling_freq(const struct device *dev, struct sensor_value *val)
+{
+	uint32_t timing;
+	struct vl53l0x_data *drv_data = dev->data;
+	int ret = VL53L0X_GetMeasurementTimingBudgetMicroSeconds(&drv_data->vl53l0x, &timing);
+	if (ret) {
+		LOG_ERR("[%s] VL53L0X_GetMeasurementTimingBudgetMicroSeconds failed", dev->name);
```

Comment:
Please use more meaningful error messages instead of just printing painfully long function names.

Reply:
I agree with that. However all the existing log messages in this driver (since its birth in Zephyr https://github.com/zephyrproject-rtos/zephyr/commit/347560459b0c55efe95e2764197a96ce22247b02) use this same form, therefore we would have to change them all for consistency ?

Reply:
If you're up for that yes, but I'd also be fine with just making your new log messages meaningful even if they're inconsistent with the old ones.